mindsdb/integrations/handlers/bard_handler/bard_handler.py

```diff
@@ -0,0 +1,103 @@
+import os
+from typing import Dict, Optional
+
+import google.generativeai as genai
+import pandas as pd
+from mindsdb.integrations.libs.base import BaseMLEngine
+from mindsdb.utilities import log
+from mindsdb.utilities.config import Config
+
+logger = log.getLogger(__name__)
+
+
+class BardHandler(BaseMLEngine):
```

Comment:
As discussed above, please rename the handler to avoid using Bard because it is not accurate.

Reply:
yes sure